transcendence-app/src/shared/db/postgres/utils.ts

```diff
@@ -21,21 +23,14 @@ export const makeQuery = async <T>(
 ): Promise<T[] | null> => {
   try {
     const res = await pool.query(query);
-    return res?.rows?.length ? (res.rows as T[]) : null;
-  } catch (e) {
-    // TODO temp solution until we implement logging
-    throw new Error(stringifyPostgresError(e));
-  }
-};
-
-const stringifyPostgresError = (err: any): string => {
-  const unkownError = 'unknown error';
-  const severity = err?.severity ?? null;
-  const code = err?.code ?? null;
-  const detail = err?.detail ?? null;
-
-  if (!severity || !code || !detail) {
-    return unkownError;
+    return res.rows;
+  } catch (err) {
+    if (
+      err instanceof DatabaseError &&
+      (err.severity === 'FATAL' || err.severity === 'PANIC')
+    ) {
+      PostgresLogger.error(err.message);
+    }
```

Comment:
Here we log database errors only for unrecoverable cases:
FATAL -> Reports an error that caused the current session to abort.
PANIC -> Reports an error that caused all sessions to abort.

We don't log errors for these cases:
ERROR -> Reports an error that caused the current command to abort.

https://www.postgresql.org/docs/current/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS